portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function getUserName(){
+    const userName =  fetch("/data").then(response => response.text()).then(name);
```

Comment:
This doesn't look like correct syntax to me - the final `then()` either returns nothing, or exactly what it receives as an input

Reply:
> This doesn't look like correct syntax to me - the final `then()` either returns nothing, or exactly what it receives as an input

Thank you for catching this error. I will correct it ASAP